.env.example

```diff
@@ -1,8 +1,12 @@
 ETHERSCAN_API_KEY=ABC123ABC123ABC123ABC123ABC123ABC1
-SEPOLIA_URL=
-ETHEREUM_URL=
+SEPOLIA_URL=https://eth-sepolia.g.alchemy.com/v2/<YOUR ALCHEMY KEY>
+ETHEREUM_URL=https://eth-mainnet.g.alchemy.com/v2/<YOUR ALCHEMY KEY>
 POLYGON_URL=https://polygon-mainnet.g.alchemy.com/v2/<YOUR API KEY>
+MUMBAI_URL=
+BASE_URL=
+BASE_GOERLI_URL=
```

Comment:
Let's fill these out with examples as well (like you did for Sepolia and Ethereum) 🤔